README.md

```diff
@@ -15,6 +15,11 @@ For more detailed information on the server and API design refer to the [wiki](h
 | NI-Digital Pattern Driver | 20.6.0 | Not Supported | Not Supported |
 | NI-DMM                    | 20.0.0 | 20.1.0        | 20.5.0        |
 | NI-FGEN                   | 20.0.0 | Not Supported | Not Supported |
+| NI-RFmx Bluetooth         | 21.5.0 | Not Supported | Not Supported |
```

Comment:
I believe 21.5 is the version we tested for all the drivers, please correct me if I'm wrong though.

Reply:
Yeah I tested with released drivers